docs/source/services.md

```diff
@@ -301,10 +338,10 @@ and taking note of the following process:
 3. On success, the reply will be a JSON model describing the user:
 
    ```json
-   { 
+   {
      "name": "inara",
      "groups": ["serenity", "guild"],
-     
+
    }
    ```
 
```

Comment:
Worth adding a section at the bottom about the nbviewer implementation? It's not very instructive, but it is a an example of something you might conceivably want to secure behind Hub auth.

* https://github.com/jupyter/nbviewer#securing-the-notebook-viewer
* https://github.com/jupyter/nbviewer/blob/master/nbviewer/providers/base.py#L94